include/roboteam_utils/BiMap.h

```diff
@@ -69,13 +69,13 @@ namespace rtt {
          * @return false If key is not found
          */
```

Comment:
I suppose these changes are an attempt to make this class compile. I saw you made an issue to do it later.

Reply:
Yes.They don't give compilation errors now but when you call the constructors somewhere they give linker errors. Still not ideal.

Reply:
They aren't used anywhere though, so it's not really important for now.